ev3dev/display.py

```diff
@@ -267,3 +277,120 @@ def update(self):
             self.mmap[:] = self._img_to_rgb565_bytes()
         else:
             raise Exception("Not supported")
+
+    def image_filename(self, filename, clear_screen=True, x1=0, y1=0, x2=None, y2=None):
+
+        if clear_screen:
+            self.clear()
+
+        filename_im = Image.open(filename)
+
+        if x2 is not None and y2 is not None:
+            return self._img.paste(filename_im, (x1, y1, x2, y2))
+        else:
+            return self._img.paste(filename_im, (x1, y1))
+
+    def line(self, clear_screen=True, x1=10, y1=10, x2=50, y2=50, line_color='black', width=1):
+        """
+        Draw a line from (x1, y1) to (x2, y2)
+        """
+
+        if clear_screen:
+            self.clear()
+
+        return self.draw.line((x1, y1, x2, y2), fill=line_color, width=width)
+
+    def circle(self, clear_screen=True, x=50, y=50, radius=40, fill_color='black', outline_color='black'):
```

Comment:
These defaults seem a bit random :laughing:

I swear, this is my last comment :wink: I need to be better about all-in-one reviews.

Reply:
yeah those are just so the user sees a full circle on the screen if they are kicking the tires and call `circle()` without passing any args